### DISCUSSION TITLE
Llava-OV video inference

### DISCUSSION THREAD

**Discussion by ehayeshaiper** [sgl-project/sglang#1330] [Q&A] [answered]
Hi, thanks for open sourcing this excellent work.
I have a concern about video inference on short (<16 frames) videos:

It seems as though we force multi-frame inference (without pooling the visual features) when the number of video frames is less than 16:
https://github.com/sgl-project/sglang/blob/12cb115d381cc19605c2fd3aa696ddf550f480de/python/sglang/srt/models/llava.py#L293

This means that the input context length is several times larger than it should be for video. Based on its descriptions it also seems as though the model interprets them as images.
Can we confirm what the correct behaviour is for short videos?
Thanks

**Comment by merrymercy** [comment, marked as answer]
fixed by https://github.com/sgl-project/sglang/pull/1346
